web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -740,8 +740,8 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
           <>
             <p>Load data accessible through HTTP(s).</p>
             <p>
-              Data must be in a text format and the HTTP(s) endpoint must be reachable by every
-              Druid process in the cluster.
+              Data must be in text, orc, or parquet format and the HTTP(s) endpoint must be
```

Comment:
nit: I don't think this is necessary to do now, but maybe next binary input format we add support for, like avro or whatever, we should just adjust this to say something like

> Data must be in a Druid supported input format ...

and perhaps linking to where-ever we describe the input formats before digging into the input source specific information.